### ISSUE TITLE
[List]: The "More" button at the end of the list records section lacks an accessible name

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents#10907]
### Describe the bug

<img width="1039" alt="Image" src="https://github.com/user-attachments/assets/1c26cfa1-ae28-46be-a589-68ee510da691" />

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-list--docs

### Reproduction steps

Set the `growing` into `Button`.
Use screen reader and focus on the More button.

### Expected Behaviour

The "More" button at the end of the list records section lacks an accessible name, making it unclear to screen reader users what action it performs.

### UI5 Web Components for React Version

2.7

### UI5 Web Components Version

2.7

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,

could you please check this issue?

Thanks,
Todor

**Comment by PetyaMarkovaBogdanova** [comment]
Similar to: #10671

**Comment by dobrinyonkov** [comment]
duplicate: https://github.com/SAP/ui5-webcomponents/issues/10671
